src/modules/position_estimator_inav/CMakeLists.txt

```diff
@@ -31,16 +31,11 @@
 #
 ############################################################################
 
-set(MODULE_CFLAGS )
-if(${OS} STREQUAL "nuttx")
-	list(APPEND MODULE_CFLAGS -Wframe-larger-than=4000)
```

Comment:
There is one catch: The stack size in the CMake file is NOT the stack of the daemon. Its the stack of the shell handler. The relationship is that the stack size provided to pthread_create() and px4_task_spawn needs to be larger than the frame size. The CMake stack size is only relevant for non-daemons, e.g. the ```pwm``` app.